pinns/pinns.c

```diff
@@ -77,6 +81,19 @@ int main(int argc, char **argv) {
       bind_user = true;
       num_unshares++;
       break;
+    case 'p':
+      unshare_flags |= CLONE_NEWPID;
```

Comment:
`unshare()` won't be enough to create a new PID namespace.

a PID namespace is created after a `fork()`

Reply:
Hah, thanks for the hint. I assume we just have to pin the non PID 1 namespace after the fork, right? (see latest implementation)